<launch>

<!--    <arg name="vicon_topic" value="/pos_vel_mocap/odom_TA"/>-->
    <arg name="imu_topic" default="/djiros/imu"/>
    <arg name="uav_topic" value="/uwb_vicon_odom"/>
    <arg name="cone_topic" value="/uwb_vicon_odom_cone"/>
    <arg name="vins_topic" value="/vins_estimator/imu_propagate"/>

    <node pkg="rnw_ros" type="pub_frames_node" name="pub_frames_node" output="screen">
        <remap from="~imu" to="$(arg imu_topic)"/>
        <remap from="~uav" to="$(arg uav_topic)"/>
        <remap from="~cone" to="$(arg cone_topic)"/>
        <remap from="~vins" to="$(arg vins_topic)"/>
    </node>

    <node pkg="rqt_multiplot" type="rqt_multiplot" name="rqt_multiplot"
          args="--multiplot-config $(find rnw_ros)/rqt_multiplot.xml --multiplot-run-all"/>

</launch>
